tests/server/blueprints/institutes/test_institute_views.py

```diff
@@ -1,7 +1,314 @@
 # -*- coding: utf-8 -*-
+import datetime
 from flask import url_for
 from scout.server.extensions import store
 
+HPO_TERMS = [
```

Comment:
Now convinced - these certainly belong as fixtures!

Reply:
😄 fixture it is!